scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala

```diff
@@ -183,6 +186,23 @@ object JobWitDuplicateOutput {
   }
 }
 
+object MetricsJob {
+  def main(cmdlineArgs: Array[String]): Unit = {
+    val (sc, args) = ContextAndArgs(cmdlineArgs)
+    val counter = ScioMetrics.counter("counter")
```

Comment:
Move metrics outside of the `main` and into the object, just in case people look at this for the example.